pkg/scaffold/util/util.go

```diff
@@ -53,20 +53,35 @@ func GetResourceInfo(r *resource.Resource, repo, domain string) (resourcePackage
 		"setting":               "k8s.io",
 		"storage":               "k8s.io",
 	}
-	resourcePath := filepath.Join("api", r.Version, fmt.Sprintf("%s_types.go", strings.ToLower(r.Kind)))
+
+	var resourcePath string
+	if isMultiGroup {
+		resourcePath = filepath.Join("apis", r.Group, r.Version, fmt.Sprintf("%s_types.go", strings.ToLower(r.Kind)))
+	} else {
+		resourcePath = filepath.Join("api", r.Version, fmt.Sprintf("%s_types.go", strings.ToLower(r.Kind)))
+	}
+
 	if _, err := os.Stat(resourcePath); os.IsNotExist(err) {
 		if domain, found := coreGroups[r.Group]; found {
 			// TODO: support apiextensions.k8s.io and metrics.k8s.io.
 			// apiextensions.k8s.io is in k8s.io/apiextensions-apiserver/pkg/apis/apiextensions
 			// metrics.k8s.io is in k8s.io/metrics/pkg/apis/metrics
 			resourcePackage := path.Join("k8s.io", "api", r.Group)
-			groupDomain := r.Group
+			if isMultiGroup {
```

Comment:
This is not true, you do not need to change `"api"` to `"apis"` for core resources.

Reply:
Sorry, your comment is not clear for me.

Reply:
Doesn't matter if you are in a multigroup or a single-group project. The path `"k8s.io/..."` is for builtin groups and the path will always be `"k8s.io/api/..."`. Just remove this conditional.